debian/jitsi-meet-web-config.postinst

```diff
@@ -212,6 +216,33 @@ case "$1" in
             fi
 
             invoke-rc.d nginx reload || true
+        elif [[ "$FORCE_OPENRESTY" = "true" && ( -z "$JVB_HOSTNAME_OLD" || "$RECONFIGURING" = "true" ) ]] ; then
+
+            # this is a reconfigure, lets just delete old links
+            if [ "$RECONFIGURING" = "true" ] ; then
+                rm -f /etc/jitsi/meet/$JVB_HOSTNAME_OLD-config.js
+            fi
+
+            # create conf
+            if [ ! -f /etc/jitsi/openresty/$JVB_HOSTNAME.conf ] ; then
+                mkdir -p /etc/jitsi/openresty
+                cp /usr/share/jitsi-meet-web-config/jitsi-meet.example /etc/jitsi/openresty/$JVB_HOSTNAME.conf
+                sed -i "s/jitsi-meet.example.com/$JVB_HOSTNAME/g" /etc/jitsi/openresty/$JVB_HOSTNAME.conf
+            fi
+
+            if [ "$CERT_CHOICE" = "$UPLOADED_CERT_CHOICE" ] ; then
+                # replace self-signed certificate paths with user provided ones
+                CERT_KEY_ESC=$(echo $CERT_KEY | sed 's/\./\\\./g')
+                CERT_KEY_ESC=$(echo $CERT_KEY_ESC | sed 's/\//\\\//g')
+                sed -i "s/ssl_certificate_key\ \/etc\/jitsi\/meet\/.*key/ssl_certificate_key\ $CERT_KEY_ESC/g" \
+                    /etc/jitsi/openresty/$JVB_HOSTNAME.conf
+                CERT_CRT_ESC=$(echo $CERT_CRT | sed 's/\./\\\./g')
+                CERT_CRT_ESC=$(echo $CERT_CRT_ESC | sed 's/\//\\\//g')
+                sed -i "s/ssl_certificate\ \/etc\/jitsi\/meet\/.*crt/ssl_certificate\ $CERT_CRT_ESC/g" \
+                    /etc/jitsi/openresty/$JVB_HOSTNAME.conf
+            fi
+
+            invoke-rc.d openresty reload || true
```

Comment:
As this reuses the nginx template and only the reload command is different. You better merge the logic with the nginx one so we do not duplicate those sed commands.